linux_os/guide/system/software/updating/ensure_GPG_keys_are_configured/rule.yml

```diff
@@ -0,0 +1,124 @@
+documentation_complete: true
+
+prodtype: sle12,sle15
+
+title: 'Ensure GPG keys are configured'
+
+description: |-
+    Most distributions use a package manager such as yum, apt, or zypper to install and 
+    update software packages. They use their own GPG keys directly or at least ship them
+    installed with the minimal system.  But third party packages signed by a different 
+    GPG key need manual handling. The operation system or installed application can be 
```

Comment:
I believe everything before `The operation system or installed application...` could be removed. It is not essential information and would be make rule description short.